scss/atoms/meter/_variables.scss

```diff
@@ -0,0 +1,20 @@
+/*
+* ========================================================================== 
+* meter variables																	  	
+* ==========================================================================
+*/
+$chassis-blue: #37bfc9;
+$meterbar-backgroundcolor: #f1f1f1;
```

Comment:
why aren't we consistent with the CSS property itself and use `background-color` instead of `backgroundcolor`?